modules/network-operator-cr.adoc

```diff
@@ -0,0 +1,94 @@
+= Network Operator custom resource
+
+The `Network` configuration in an `operator.openshift.io` Custom Resource (CR)
+object stores the configuration settings for the Network Operator.
+
+The following parameters are set in the `Network.config.openshift.io` CR and
+copied to the `Network.operator.openshift.io` CR by the Network Operator:
+
+* `clusterNetwork.cidr`
+* `clusterNetwork.hostPrefix`
+* `serviceNetwork`
+* `networkType`
+
+The following CR displays the default configuration for the Network Operator and
+explains both the parameters you can configure and valid parameter values:
+
+.Network.operator.openshift.io
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+spec:
+  clusterNetworks:
+  - cidr: 10.128.0.0/14 <1>
+    hostPrefix: 23 <2>
+  serviceNetwork: 172.30.0.0/16 <3>
+  defaultNetwork: <4>
+    type: OpenShiftSDN <5>
+    openshiftSDNConfig: <6>
+      mode: NetworkPolicy <7>
+      mtu: 1450 <8>
+      vxlanPort: 4789 <9>
+  deployKubeProxy: false
+  kubeProxyConfig: <10>
+    bindAddress: 0.0.0.0 <11>
+    iptablesSyncPeriod: 30s <12>
+    proxyArguments:
+      iptables-min-sync-period: <13>
+      - 30s
+----
+
+<1> `cidr`: A block of IP addresses from which pod IPs are allocated.
+{product-title} uses the Classless Inter-Domain Routing (CIDR) standard to
+allocate IP addresses. The 'OpenShiftSDN' network plugin supports multiple
+ClusterNetworks. The address blocks for multiple ClusterNetworks must not
+overlap. This parameter is set in the `config.openshift.io` object.
+<2> `hostPrefix`: 
+<3> `serviceNetwork`: A block of IP addresses for services. `OpenShiftSDN`
+allows only one serviceNetwork block. This parameter is set in the
+`config.openshift.io` object.
+<4> `defaultNetwork`: The header for the parameters that specify the
+default pod network.
+<5> `type`: The Network plugin that is to be deployed. `OpenShiftSDN` is the
+only plugin supported in {product-title} {product-version}. This parameter is
+set in the `config.openshift.io` object.
+<6> `openshiftSDNConfig`: Specify only if you want to override part of the
+{product-title} SDN configuration.
+<7> `mode`: Configures the isolation mode for `OpenShiftSDN`. The allowed values
+are `Multitenant`, `Subnet`, or `NetworkPolicy`. The default value is
+`NetworkPolicy`.
+<8> `mtu`: MTU for the VXLAN overlay network. This is normally configured
+automatically, but if the nodes in your cluster do not all use the same MTU,
+then you must set this explicitly to 50 less than the smallest node MTU value.
+<9> `VXLANPort`: The port to use for all VXLAN packets. The default value is
+4789. You should not need to change this unless you are running in a virtualized
+environment where some nodes are already part of another VXLAN network.
+<10> `kubeProxyConfig`: The parameters in this section specify the
+`kube-proxy` configuration. If you do not specify the parameter values, the
+Network Operator chooses sensible defaults.
```

Comment:
s/chooses sensible defaults/applies the default parameter values.
What's a sensible default? When do I know if I want something other than a sensible default? Isn't this one of the only values that can be changed after installation? You should point that out.